kong/runloop/plugins_iterator.lua

```diff
@@ -86,7 +87,7 @@ local function get_next(self)
   local ctx = self.ctx
 
   -- load the plugin configuration in early phases
-  if self.access_or_cert_ctx then
+  if self.load_configuration then
```

Comment:
I strongly recommend `load_plugins` or `load_plugins_for_request` instead (to mirror the `ctx.plugins_for_request` variable). `load_configuration` is too vague (too many things are considered "configuration": Admin API entities, `kong.conf`, `kong.yml`, etc...).